.agent/skills/nextjs-app-router/SKILL.md

```
@@ -0,0 +1,98 @@
+
+// error.tsx — Shown on error
+'use client';
+export default function Error({ reset }: { reset: () => void }) {
+  return <button onClick={reset}>Retry</button>;
+}
+```
+
+## Error Handling
+
+| Error | Cause | Solution |
+|-------|-------|----------|
```

markdownlint-cli2 (0.20.0)

[warning] Table column style
Table pipe is missing space to the right for style "compact"

(MD060, table-column-style)